packages/statemanager/README.md

```diff
@@ -62,24 +62,38 @@ First, a simple example of usage:
 ```typescript
 import { Account, Address } from '@ethereumjs/util'
 import { EthersStateManager } from '@ethereumjs/statemanager'
-import { ethers } from 'ethers'
 
-const provider = new ethers.providers.JsonRpcProvider('https://path.to.my.provider.com')
+const provider = 'https://path.to.my.provider.com'
 const stateManager = new EthersStateManager({ provider, blockTag: 500000n })
 const vitalikDotEth = Address.fromString('0xd8da6bf26964af9d7eed9e03e53415d37aa96045')
 const account = await stateManager.getAccount(vitalikDotEth)
 console.log('Vitalik has a current ETH balance of ', account.balance)
 ```
 
-The `EthersStateManager` can be be used with an `ethers` `JsonRpcProvider` or one of its subclasses. Instantiate the `VM` and pass in an `EthersStateManager` to run transactions against accounts sourced from the provider or to run blocks pulled from the provider at any specified block height.
+The `EthersStateManager` can be be used with any JSON-RPC provider that supports the `eth` namespace. Instantiate the `VM` and pass in an `EthersStateManager` to run transactions against accounts sourced from the provider or to run blocks pulled from the provider at any specified block height.
 
 **Note:** Usage of this StateManager can cause a heavy load regarding state request API calls, so be careful (or at least: aware) if used in combination with an Ethers provider connecting to a third-party API service like Infura!
 
 ### Points on usage:
 
+#### Instantiating the EVM
+
+In order to have a fully functioning EVM instance, you must also instantiate the `EthersStateManager` and the `RpcBlockChain` and use that when instantiating your EVM as below:
```

Comment:
I would like a clarification here what "fully functioning" means. E.g: "In order to have a fully functioning EVM instance, which also supports the `BLOCKHASH` opcode, you must also initialize the..."